FirebaseRemoteConfig/Sources/RCNConfigDBManager.h

```diff
@@ -110,6 +111,15 @@ typedef void (^RCNDBLoadCompletion)(BOOL success,
 /// Insert or update the data in Personalization config.
 - (BOOL)insertOrUpdatePersonalizationConfig:(NSDictionary *)metadata fromSource:(RCNDBSource)source;
 
+/// Insert rollout metadata in rollout table.
+/// @param key        Key indicating whether rollout metadata is fetched or active and defined in
+/// RCNConfigDefines.h.
+/// @param value      The value that rollout metadata array.
```

Comment:
Reword

Reply:
@danasilver, @ddnan have the fix here: https://github.com/firebase/firebase-ios-sdk/pull/1250, mind to take a look?